ui/test/nuxeo-date.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture, flush } from '@nuxeo/testing-helpers';
+import moment from '@nuxeo/moment';
+import '../widgets/nuxeo-date.js';
+import '../widgets/nuxeo-tooltip.js';
+
+suite('nuxeo-date', async () => {
+  test('Should hide the nuxeo-tooltip if its the same format', async () => {
+    const element = await fixture(
+      html`
+        <nuxeo-date datetime=${moment()} format="LLL" tooltipFormat="LLL"></nuxeo-date>
+      `,
+    );
+    await flush();
+
+    const tooltip = element.shadowRoot.querySelector('nuxeo-tooltip');
+    expect(tooltip.hidden).to.be.true;
+  });
+  test('Should show the nuxeo-tooltip if its a different format', async () => {
+    const element = await fixture(
+      html`
+        <nuxeo-date datetime=${moment()} format="relative" tooltipFormat="LLL"></nuxeo-date>
+      `,
+    );
+    await flush();
+
+    const tooltip = element.shadowRoot.querySelector('nuxeo-tooltip');
+    expect(tooltip.hidden).to.be.false;
+  });
+});
```

Comment:
Since we are relying on behavior format functions that also take into account the global configs `Nuxeo.UI.config.`, it would be nice if we could also add those test scenarios.

Reply:
done